tool/src/java/org/sakaiproject/gradebookng/GradebookNgApplication.properties

```diff
@@ -17,6 +17,9 @@ label.total=Total:
 label.due=Due:
 label.noduedate=-
 
+label.coursegrade.released=Course grade has been released to students.
+label.coursegrade.notreleased=Course grade has not been released to students.
+
```

Comment:
These have fullstops but the others don't. Maybe we should add fullstops to all of the labels that are used like this? Or none at all?